addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -160,19 +154,31 @@ class Composer extends Component {
      *
      * @private
      */
-    async _postMessage() {
+    _postMessage() {
         // TODO: take suggested recipients into account
         this.storeDispatch('postMessage', this.props.composerLocalId, {
-            htmlContent: this._textInputRef.comp.getHtmlContent(),
+            content: this._textInputRef.comp.getContent(),
```

Comment:
don't think you need this, since this is already in store (`composer.textInputContent`)

Reply:
Done